crates/volta-core/src/tool/package_global/uninstall.rs

```diff
@@ -72,20 +78,18 @@ fn remove_config_and_shim(bin_name: &str, pkg_name: &str) -> Fallible<()> {
 /// all the binaries installed by the given package.
 fn binaries_from_package(package: &str) -> Fallible<Vec<String>> {
     let bin_config_dir = volta_home()?.default_bin_dir();
-    if bin_config_dir.exists() {
-        dir_entry_match(&bin_config_dir, |entry| {
-            let path = entry.path();
-            if let Ok(config) = BinConfig::from_file(path) {
-                if config.package == package {
-                    return Some(config.name);
-                }
+
+    dir_entry_match(&bin_config_dir, |entry| {
+        let path = entry.path();
+        if let Ok(config) = BinConfig::from_file(path) {
+            if config.package == package {
+                return Some(config.name);
             }
-            None
-        })
-        .with_context(|| ErrorKind::ReadBinConfigDirError {
-            dir: bin_config_dir.to_owned(),
-        })
-    } else {
-        Ok(Vec::new())
-    }
+        }
+        None
+    })
+    .with_context(|| ErrorKind::ReadBinConfigDirError {
+        dir: bin_config_dir.to_owned(),
+    })
+    .accept_error_as_default_if(|e| e.is_not_found_error_kind())
```

Comment:
Same comment here about `dir_entry_match` and `volta_core::fs::ok_if_not_found`

Reply:
Done